pkg/pool-stable/contracts/ComposableStablePool.sol

```diff
@@ -979,6 +979,40 @@ contract ComposableStablePool is
         return currentInvariant.divDown(actualTotalSupply);
     }
 
+    function _beforeProtocolFeeCacheUpdate() internal override {
+        // The `getRate()` function depends on the actual supply, which in turn depends on the cached protocol fee
+        // percentages. Changing these would therefore result in the rate changing, which is not acceptable as this is a
+        // sensitive value.
+        // Because of this, we pay any due protocol fees *before* updating the cached rates, making it so that the new
+        // percentages only affect future operation of the Pool, and not past fees. As a result, `getRate()` is
+        // unaffected by the cached protocol fee percentages changing.
```

Comment:
```suggestion
        // Because of this, we pay any due protocol fees *before* updating the cache, making it so that the new
        // percentages only affect future operation of the Pool, and not past fees. As a result, `getRate()` is
        // unaffected by the cached protocol fee percentages changing.
```

"Cached rates" reads as the token rates for yield fees to me. Let's just say "cache" as it's clear which cache we're talking about from the function name.